presto-main/src/main/java/com/facebook/presto/server/InternalCommunicationConfig.java

```diff
@@ -179,4 +180,17 @@ public InternalCommunicationConfig setTaskCommunicationProtocol(CommunicationPro
         this.taskCommunicationProtocol = taskCommunicationProtocol;
         return this;
     }
+
+    public CommunicationProtocol getServerInfoCommunicationProtocol()
+    {
+        return serverInfoCommunicationProtocol;
+    }
+
+    @Config("internal-communication.server-info-communication-protocol")
+    @ConfigDescription("Set server info communication protocol")
```

Comment:
When this set to THRIFT, does it mean only `state` call will be using thrift, while other call still use HTTP?

Reply:
@highker : I see this comment is marked as resolved but didn't see the answer? :)

Reply:
I have update the desc: https://github.com/prestodb/presto/pull/13894/commits/c7022d6b45420be36cf772b5fef9b3fdbf9866dd#diff-2163af643220b1ee3dfa225e9dc42e49R190